server/.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "root": true,
+  "parserOptions": {
+    "ecmaVersion": 12,
+    "sourceType": "module"
+  },
+  "extends": ["eslint:recommended", "prettier"],
+  "env": {
+    "es2021": true,
+    "node": true
+  },
+  "rules": {}
```

Comment:
Can you put the rules, so when husky is set up it should not allow to be pushed with errors.
```
"rules": {
        "indent": 
        [
            "error",
            2
        ],
        "quotes": [
            "error",
            "single"
        ],
        "semi": [
            "error",
            "always"
        ],
        "no-console": [
            "error"
        ],
}
```

Reply:
Yeah, maybe we if add a logging tool like winston. It is not necessary right now